scripts/release-docker.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Author's Note: Because the structure of this repo is inconsistent (Dockerfiles and build contexts placed willy-nilly)
+# we have to whitelist and configure image builds individually because each one is going to be slightly different. 
+# This is needed as opposed to trusting the structure of the each project to be consistent for every deployable. 
+
+set +x
+
+CLEAR='\033[0m'
+RED='\033[0;31m'
+# Array of valid service names
+VALID_SERVICES=('daemon' 'bot')
+
+function usage() {
+  if [ -n "$1" ]; then
+    echo -e "${RED}👉  $1${CLEAR}\n";
```

Comment:
Can you use unicode instead of emoji? This will just appear as a box for non OSX users.

Reply:
We use emojis heavily in the new Coda CLI, I think this is consistent with the style we've laid out thus far.